### PR TITLE
remove beta version checks from console

### PR DESCRIPTION
**Pull request by rikinsk**
### Affected components

- [ ] Server
- [x] Console
- [ ] CLI
- [ ] Docs
- [ ] Community Content
- [ ] Build System
- [ ] Tests
- [ ] Other (list it)

#### Catalog upgrade

Does this PR change Hasura Catalog version?
- [ ] No
- [ ] Yes
  - [ ] Updated docs with SQL for downgrading the catalog 

#### Metadata

Does this PR add a new Metadata feature?
- [ ] No
- [ ] Yes
  - Does `run_sql` auto manages the new metadata through schema diffing?
    - [ ] Yes
    - [ ] Not required
  - Does `run_sql` auto manages the definitions of metadata on renaming?
    - [ ] Yes
    - [ ] Not required
  - Does `export_metadata`/`replace_metadata` supports the new metadata added?
    - [ ] Yes
    - [ ] Not required

#### GraphQL
- [ ] No new GraphQL schema is generated
- [ ] New GraphQL schema is being generated:
   - [ ] New types and typenames are correlated
   
   

#### Breaking changes

- [ ] No Breaking changes
- [ ] There are breaking changes:

  1. Metadata API

     Existing `query` types:
     - [ ] Modify `args` payload which is not backward compatible
     - [ ] Behavioural change of the API
     - [ ] Change in response `JSON` schema
     - [ ] Change in error code
     

  2. GraphQL API

     Schema Generation:
     
     
     - [ ] Change in any `NamedType`
     - [ ] Change in table field names
     

     Schema Resolve:-
     
     - [ ] Change in treatment of `null` value for any input fields 
     

  3. Logging

     - [ ] Log `JSON` schema has changed
     - [ ] Log `type` names have changed

### CODE REVIEW

File: console/src/components/Main/Main.js [hunk 1]
```diff
@@ -105,12 +91,6 @@ class Main extends React.Component {
     }
   }
```
Comment (wawhal) [inline console/src/components/Main/Main.js]: I think we need to `fetchServerConfig` for JWT decoder.

File: console/src/components/Services/EventTrigger/EventActions.js [hunk 2]
```diff
@@ -107,21 +106,14 @@ const loadPendingEvents = () => (dispatch, getState) => {
             '*',
             { name: 'logs', columns: ['*'], order_by: ['-created_at'] },
           ],
-          where: { delivered: false, error: false, tries: 0 },
+          where: { delivered: false, error: false, tries: 0, archived: false },
```
Comment (wawhal) [inline console/src/components/Services/EventTrigger/EventActions.js]: What is this?

File: console/src/components/Main/Main.js (hunk 1 above)
Comment (rikinsk) [reply console/src/components/Main/Main.js]: Yep. I moved it to line 69. Removed unnecessary fn

File: console/src/components/Services/EventTrigger/EventActions.js (hunk 2 above)
Comment (rikinsk) [reply console/src/components/Services/EventTrigger/EventActions.js]: This was some event trigger server refactor related thing. Event data can be archived so we need to add thid flag to ensure we fetch only non archived data

Comment (rikinsk) [reply console/src/components/Services/EventTrigger/EventActions.js]: it was initially added after version check below

**Pull request merged**